src/examples/dropdowns/SelectSize.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Dropdown, Field, Menu, Item, Select, Label } from '@zendeskgarden/react-dropdowns';
+
+interface IItem {
+  label: string;
+  value: string;
+}
+
+const items = [
+  { label: 'Cactus', value: 'item-1' },
+  { label: 'Orchid', value: 'item-2' },
+  { label: 'Succulent', value: 'item-3' }
+];
+
+const StyledField = styled(Field)`
+  * {
+    /* stylelint-disable-next-line declaration-no-important */
+    appearance: none !important;
+  }
+`;
+
+const Example = () => {
+  const [selectedItem, setSelectedItem] = useState(items[0]);
```

Comment:
This is a single state that powers both dropdowns. Feels a bit strange, so I'm going to have state for each of the dropdown sizes.